src/components/Modal/ExternalLinkWarningModal/ExternalLinkWarningModal.tsx

```diff
@@ -37,7 +35,7 @@ function ExternalLinkWarningModal() {
   const { isWarningModalOpen, href } = warningModalState
   const handleDismiss = () => setWarningModalState((state) => ({ ...state, isWarningModalOpen: false }))
   const handleContinue = useCallback(() => {
-    openInNewTab(href)
+    openUrl(href)
```

Comment:
I think this should be `openUrl(href, true)`

Reply:
By default it is true!

![image](https://github.com/decentraland/governance/assets/45410089/d2769970-66df-4e39-9314-b04a28466f1a)